src/file/file.service.ts

```diff
@@ -220,30 +233,15 @@ export class FileService {
    * @param queryFile - the partial file containing the conditions.
    * @returns the pure query for extracting from the database.
    */
-  private static extractQuery(folderID: string | null, ownerID: string | null, queryFile?: Partial<IFile>): Partial<IFile> {
-    const parent = folderID ? new ObjectID(folderID) : null;
-    let query : Partial<IFile> = {};
-
-    // Create the query using the partial file,
+  private static extractQuery(queryFile?: Partial<IFile>): Partial<IFile> {
```

Comment:
Please consider refactoring to this function's documentation and naming.

Reply:
right, I forgot to update it